update/Pos/Update/Configuration.hs

```diff
@@ -92,18 +89,7 @@ curSoftwareVersion = SoftwareVersion ourAppName (ccApplicationVersion updateConf
 -- from @Cabal@ was used to access to a build's host machine @OS@ and @Arch@itecture
 -- information.
 currentSystemTag :: SystemTag
-currentSystemTag =
-    $(do let tag :: SystemTag
-             tag = SystemTag (toText (osHelper buildOS ++ archHelper buildArch))
-             st :: Either Text ()
-             st = checkSystemTag tag
-             color c s = "\n" <> colorize c s <> "\n"
-         case st of Left e -> error . color Red . T.concat $
-                                  ["Current system tag could not be calculated: ", e]
-                    Right () -> do runIO . putStrLn . color Blue . T.concat $
-                                       ["Current system tag is: ", show tag]
-                                   TH.lift tag
-     )
+currentSystemTag = SystemTag (toText (osHelper buildOS ++ archHelper buildArch))
```

Comment:
The template Haskell would fail if the system tag is too long (`checkSystemTag`).

Is this really necessary?

Cons:
- it causes linker problems
- it's template Haskell

Pros:
- system tag is guaranteed to be short enough

So what? If the system tag is too long, verification will fail at run time.

Reply:
Related note: can anyone tell me why a system tag is even necessary? Who cares what platform the software is running on?

Reply:
Is it because update proposals version the _software_, rather than the _protocol_?

Reply:
this was always here for "one day when we package cardano for linux distros" which might be in 3 years so let's have the complexity then :)